vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/AndroidDevice.kt

```diff
@@ -37,7 +37,7 @@ import java.util.*
 import kotlin.coroutines.CoroutineContext
 
 class AndroidDevice(val ddmsDevice: IDevice,
-                    private val serialStrategy: SerialStrategy = SerialStrategy.AUTOMATIC) : Device, CoroutineScope {
+                    private val serialStrategy: SerialStrategy = SerialStrategy.DDMS) : Device, CoroutineScope {
```

Comment:
This is settable by Marathonfile. Don't see any reason to change the sourcecode for this - just use the Marathonfile

Reply:
Thanks for the input, but as far as I know there is no such parameter in that version of the Marathon project. Is it present in the latest version of the main project?

Reply:
@Malinskiy I was wrong. I will rollback this PR